satflow/models/pl_metnet.py

```diff
@@ -7,6 +7,8 @@
 from pl_bolts.optimizers.lr_scheduler import LinearWarmupCosineAnnealingLR
 from metnet import MetNet
 
+head_to_module = {"identity": nn.Identity()}
```

Comment:
This is the simplest approach I could think of to map the `head` parameter to a module, but happy to refactor if you prefer something different

Reply:
I like this approach!